FirebaseAnalyticsWrapper/Public/FirebaseAnalytics/FIRAnalytics+AppDelegate.h

```diff
@@ -0,0 +1,62 @@
+#import <Foundation/Foundation.h>
+
+#import "FIRAnalytics.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Provides App Delegate handlers to be used in your App Delegate.
+ *
+ * To save time integrating Firebase Analytics in an application, Firebase Analytics does not
+ * require delegation implementation from the AppDelegate. Instead this is automatically done by
+ * Firebase Analytics. Should you choose instead to delegate manually, you can turn off the App
+ * Delegate Proxy by adding FirebaseAppDelegateProxyEnabled into your app's Info.plist and setting
+ * it to NO, and adding the methods in this category to corresponding delegation handlers.
+ *
+ * To handle Universal Links, you must return YES in
+ * [UIApplicationDelegate application:didFinishLaunchingWithOptions:].
+ */
+@interface FIRAnalytics (AppDelegate)
```

Comment:
I wonder why we need these headers in the repo now?

Reply:
To make them visible to Objective C SPM clients.

Reply:
I curious now how it works... I thought the headers from the .zip should be visible.

Reply:
It might need a feature request. I think SPM's design is primarily focused on supporting importing modules and not individual headers.

Reply:
Did you have to create the headers manually? I wonder how we will make sure they are up to date in the long run.

Reply:
We'll either need to coordinate a SPM fix or add a step to the release process to sync the headers to the repo.

Reply:
I see. Yeah, maybe copybara may be useful here.